student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person_class'
+
+class Student < Person
+  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
+    super(name, age, parent_permission)
```

Comment:
- **[OPTIONAL]** Kindly assign `parent_permission` argument correctly.

- **[OPTIONAL]** Kindly arrange your parameter for the super method.

  ```
  super(age, name, parent_permission: parent_permission)
  ```